src/modules/Remote.rb

```diff
@@ -131,54 +148,29 @@ def Propose
     # Read the current status
     # @return true on success
     def Read
-      # are the proper services enabled in xinetd?
-      xinetd_conf = SCR.Read(path(".etc.xinetd_conf.services")) || {}
-      xinetd_vnc1_enabled = xinetd_conf.any? { |c| c["service"] == "vnc1" && c["enabled"] }
-
-      log.info "VNC1 enabled: #{xinetd_vnc1_enabled}"
+      display_manager_remote_access = SCR.Read(path(".sysconfig.displaymanager.DISPLAYMANAGER_REMOTE_ACCESS")).to_s == "yes"
 
+      xdm        = Service.Enabled(XDM_SERVICE_NAME)
       vncmanager = Service.Enabled(VNCMANAGER_SERVICE)
-      vnc_service_running = xinetd_vnc1_enabled || vncmanager
+      xvnc       = Service.Enabled(XVNC_SERVICE)
+      xvnc_novnc = Service.Enabled(XVNC_NOVNC_SERVICE)
```

Comment:
is this variable ever used?

Reply:
ah yes it is ...

Reply:
Few lines below: `EnableWebVnc() if xvnc_novnc`

Reply:
btw ```NOVNC``` in name is really funny when setting vnc access, so what is this service responsible for?

Reply:
I did not name it. :-)
https://github.com/novnc/noVNC

It is VNC viewer written in javascript and HTML5. The service serves the web page with the viewer, so the remote desktop can be optionally accessed using browser.